### DISCUSSION TITLE
table 表格合并选择复选框怎么可以让他出现一次呢？

### DISCUSSION THREAD

**Discussion by cangsnage** [ant-design/ant-design#45921] [General]
### What problem does this feature solve?

table 我做了合并表格处理，但是我有个多选的操作，表格会以为我这样会有多个数据出现多个复选框

### What does the proposed API look like?

可以自己定义复选框
![f59ae358e9fcb631ca14bf16a8f694d](https://github.com/ant-design/ant-design/assets/25116493/0e37a858-4b8a-460a-bced-4417a0e90398)

**Comment by yoyo837** [comment]
<img width="1501" alt="image" src="https://github.com/ant-design/ant-design/assets/6134547/bc61fcb4-a6b5-4016-805a-bb6f17518577">

https://github.com/ant-design/ant-design/pull/21711
